unity-renderer/Assets/Scripts/MainScripts/DCL/Environment/Factories/ServiceLocatorFactory/ServiceLocatorFactory.cs

```diff
@@ -112,13 +112,12 @@ public static ServiceLocator CreateDefault()
 
             result.Register<IFriendsController>(() =>
             {
-                // TODO (NEW FRIEND REQUESTS): remove when the kernel bridge is production ready
+                // TODO: remove when the all the friendship responsibilities are migrated to unity
```

Comment:
```suggestion
                // TODO: remove when all the friendship responsibilities are migrated to unity
```
What still is missing?

Reply:
We still use `use-social-client` which fallbacks to the WebInterface flow, for example for retrieval of friends with direct messages